.github/actions/install-python-and-package/action.yml

```diff
@@ -13,18 +13,23 @@ runs:
   using: "composite"
   steps:
     - name: Set up Python ${{ inputs.python-version }}
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@v4
       with:
         python-version: ${{ inputs.python-version }}
-        cache: 'pip'
-        cache-dependency-path: setup.cfg
+    
+    - uses: actions/cache@v3
+      with:
+        path: ${{ env.pythonLocation }}
+        key: ${{ env.pythonLocation }}-${{ hashFiles('setup.cfg') }}
+    
     - name: Python info
       shell: bash {0}
       run: |
         which python3
         python3 --version
+    
     - name: Upgrade pip and install dependencies
       shell: bash {0}
       run: |
         python3 -m pip install --upgrade pip setuptools wheel
-        python3 -m pip install .[${{ inputs.extras-require }}]
+        python3 -m pip install -e .[${{ inputs.extras-require }}]
```

Comment:
Fine either way of course. Is there any advantage of installing 'editable' like this in this case as part of CI?

Reply:
I did this to avoid caching the dianna install.